src/settings.ts

```diff
@@ -41,18 +57,6 @@ export class LinkRangeSettingTab extends PluginSettingTab {
 					postProcessorUpdate(this.app)
 				}));
 
-		new Setting(containerEl)
-			.setName('Alt Text Format')
```

Comment:
I know there are other people using the plugin so obviously this would modify their existing settings by removing this, not sure if there's a way we could avoid that or not. A changelog might work. Not a huge deal though I don't think, as it's easy enough for people to go in and create the settings the way they want after updating.

Reply:
Okay, I put in some code to migrate the old altFormat to the new data format.